src/check-nsfw.js

```diff
@@ -69,6 +69,7 @@ async function checkNSFW(message, urls) {
 				// aggressive NSFW check
 				// flag the GIF if ANY NSFW frame is found
 				if (frameClassification.className === 'Porn' || frameClassification.className === 'Hentai' || frameClassification.className === 'Sexy') {
+					if (frameClassification.probability < 0.8) return;  // if low confidence then do nothing
```

Comment:
Doing a return in the loop will exit the loop and not check further frames, `continue` should be used instead. Also Should probably check the confidence before checking the type?

Reply:
Fixed!